src/layout/Application/components/AddAndEditEnvBind/index.tsx

```diff
@@ -30,11 +34,14 @@ class EnvBindPlanDialog extends Component<Props, State> {
     this.field = new Field(this);
     this.state = {
       loading: false,
+      isEdit: false,
     };
   }
 
   componentDidMount() {
     this.loadDeliveryTargets();
+    this.getIsEdit();
+    this.setFields();
```

Comment:
`setState` is an asynchronous function,  and reading it immediately after being set may not get values.